lib/ansible/modules/database/mongodb/mongodb_replicaset.py

```diff
@@ -0,0 +1,412 @@
+#!/usr/bin/python
+
+# (c) 2018, Rhys Campbell <rhys.james.campbell@googlemail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: mongodb_replicaset
+short_description: Initialises a MongoDB replicaset before authentication has been turned on and then validates the configuration when it has been turned on.
+description:
+    - Initialises a MongoDB replicaset before authentication has been turned on and then validates the configuration when it has been turned on.
+    - Validation takes the form of replicaset name validation only at present.
+    - This may change in the future to involve adding  and removing members.
+author: Rhys Campbell (@rhysmeister)
+version_added: "2.8"
+options:
+    login_user:
+        description:
+            - The username used to authenticate with
+        required: false
+    login_password:
+        description:
+            - The password used to authenticate with
+        required: false
+    login_database:
+        description:
+            - The database where login credentials are stored
+        required: false
+        default: "admin"
+    login_host:
+        description:
+            - The MongoDB hostname to login to
+        required: false
+        default: "localhost"
+    login_port:
+        description:
+            - The MongoDB port to login to
+        required: false
+        default: 27017
+    replica_set:
+        description:
+            - Replicaset name
+        required: false
+        default: rs0
+    members:
+        description:
+            - An comma sepaeated list consisting of the replicaset members.
```

Comment:
This line should begin "A comma-separated list" ("a" not "an", add a hyphen to the compound adjective, typo in "separated").